pipeline/frontend/yaml/error.go

```diff
@@ -16,6 +16,11 @@ package yaml
 
 import "errors"
 
+var (
+	ErrUnsuportedVersion = errors.New("unsuported yaml version detected")
+	ErrMissingVersion    = errors.New("missing yaml version")
```

Comment:
Maybe `pipeline config` or `workflow config` instead of yaml?

```suggestion
	ErrUnsuportedVersion = errors.New("unsuported pipeline config version detected")
	ErrMissingVersion    = errors.New("missing pipeline config version")
```